### PR TITLE
add test for container push on Katello 4.14+

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: bats/fb-katello-container.bats [hunk 1]
```diff
@@ -13,3 +13,13 @@ load fixtures/content
   CONTAINER_PULL_LABEL=$(echo "${ORGANIZATION_LABEL}-${PRODUCT_LABEL}-${CONTAINER_REPOSITORY_LABEL}"| tr '[:upper:]' '[:lower:]')
   podman pull "${HOSTNAME}/${CONTAINER_PULL_LABEL}"
 }
+
+@test "push container to katello" {
+  tContainerPushSupported
+  tPackageExists podman || tPackageInstall podman
+  podman login "${HOSTNAME}" -u admin -p changeme
+  CONTAINER_PULL_LABEL=$(echo "${ORGANIZATION_LABEL}-${PRODUCT_LABEL}-${CONTAINER_REPOSITORY_LABEL}"| tr '[:upper:]' '[:lower:]')
+  CONTAINER_PUSH_LABEL=$(echo "${ORGANIZATION_LABEL}/${PRODUCT_LABEL}/${CONTAINER_REPOSITORY_LABEL}-bats-$(date -u '+%s')"| tr '[:upper:]' '[:lower:]')
```
Comment (evgeni) [inline bats/fb-katello-container.bats:22]: Yes, you need to pull with `ORG-PRODUCT-REPO` but push with `ORG/PRODUCT/REPO`.

@ianballou this hurts :crying_cat_face:

File: bats/foreman_helper.bash [hunk 2]
```diff
@@ -53,6 +53,13 @@ tSkipIfNewerThan45() {
   fi
 }
 
+tContainerPushSupported() {
+  KATELLO_VERSION=$(tKatelloVersion)
+  if ! tIsVersionNewer "${KATELLO_VERSION}" 4.13; then
```
Comment (evgeni) [inline bats/foreman_helper.bash:58]: That's based on https://github.com/Katello/katello/pull/10952 -- is that a correct assumption?

File: bats/fb-katello-container.bats (hunk 1 above)
Comment (ehelms) [reply bats/fb-katello-container.bats:22]: Aye it does - @ianballou is looking into whether we could switch to a `/` based naming scheme which is more common within the container world.

Comment (evgeni) [reply bats/fb-katello-container.bats:22]: interestingly, pushed stuff is then also only pull-able with `/` -- so `-` only applies to synced stuff? confusing!

**Review by ianballou — COMMENTED** [review]
This looks fine at a glance, was it tested on a live system?

File: bats/foreman_helper.bash (hunk 2 above)
Comment (ianballou) [reply bats/foreman_helper.bash:51]: That PR is just the beginning, I need to check but I think 4.14 would be better.

File: bats/fb-katello-container.bats (hunk 1 above)
Comment (evgeni) [reply bats/fb-katello-container.bats:22]: nah, let's keep as close to the default config as possible

**Review by ianballou — APPROVED** [review]
Looks like there was just enough of the container push feature implemented for this test to work in 4.13, nice. Ack!

File: bats/foreman_helper.bash (hunk 2 above)
Comment (ianballou) [reply bats/foreman_helper.bash:51]: @evgeni I confirmed that Katello 4.14 and higher should be used here.

**Review by ianballou — CHANGES_REQUESTED** [review]

File: bats/foreman_helper.bash (hunk 2 above)
Comment (evgeni) [reply bats/foreman_helper.bash]: done!

**Review by ianballou — APPROVED** [review]
Looks good!

**Pull request merged**